src/components/ControlInput.vue

```diff
@@ -601,6 +609,23 @@ export default {
                 list = list.concat(commandList);
             }
 
+            if (opts.settings) {
+                let out = {};
+                let base = [];
+                settingTools.buildTree(out, base, state.getSetting('settings'), false);
+                settingTools.buildTree(out, base, state.getSetting('user_settings'), true);
+
+                let settingList = [];
+                Object.keys(out).forEach((setting) => {
+                    settingList.push({
```

Comment:
Any reason not to push directly to `list` here?

Reply:
I did it this way to match how the users/buffers are done